hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/recon/TestReconTasks.java

```diff
@@ -219,9 +230,10 @@ public void testEmptyMissingContainerDownNode() throws Exception {
     ContainerManager scmContainerManager = scm.getContainerManager();
     ReconContainerManager reconContainerManager =
         (ReconContainerManager) reconScm.getContainerManager();
+    int previousContainerCount = reconContainerManager.getContainers().size();
     ContainerInfo containerInfo =
         scmContainerManager
-            .allocateContainer(RatisReplicationConfig.getInstance(ONE), "test");
+            .allocateContainer(RatisReplicationConfig.getInstance(ONE), "testEmptyMissingContainer");
```

Comment:
If you are increasing datanodes, then better to keep replication factor also as THREE

Reply:
I tried changing it to HddsProtos.ReplicationFactor.THREE but it seems having problem with number of pipelines..
java.io.IOException: Could not allocate container. Cannot get any matching pipeline for replicationConfig: RATIS/THREE, State:PipelineState.OPEN

	at org.apache.hadoop.hdds.scm.container.ContainerManagerImpl.allocateContainer(ContainerManagerImpl.java:202)
	at org.apache.hadoop.ozone.recon.TestReconTasks.testEmptyMissingContainerDownNode(TestReconTasks.java:236)
	at java.lang.reflect.Method.invoke(Method.java:498)

Reply:
Ok, this might be because of not meeting the criteria of sufficient healthy nodes because default `minRatisVolumeSizeBytes` is 1 GB and `containerSizeBytes` is 5GB. For test case it is okay then to use `ReplicationFactor.ONE`